chia/wallet/wallet_state_manager.py

```diff
@@ -275,17 +262,13 @@ async def create(
     def get_public_key_unhardened(self, index: uint32) -> G1Element:
         return master_sk_to_wallet_sk_unhardened(self.private_key, index).get_g1()
 
-    async def get_keys(self, puzzle_hash: bytes32) -> Optional[Tuple[G1Element, PrivateKey]]:
+    async def get_private_key(self, puzzle_hash: bytes32) -> PrivateKey:
         record = await self.puzzle_store.record_for_puzzle_hash(puzzle_hash)
         if record is None:
-            raise ValueError(f"No key for this puzzlehash {puzzle_hash})")
+            raise ValueError(f"No key for puzzle hash: {puzzle_hash.hex()}")
```

Comment:
we have a custom `__str__` for `bytes32` that always prints it as hex

Reply:
Yeah right but after one of your rust optimizations we have `bytes32` objects which are not really `bytes32` objects, right? Thats why i recently started to just `hex` them because the byte representation looks pretty ugly in logs :)